apps/Tables/pages/InventoryTable.ejs

```diff
@@ -51,12 +51,51 @@
 
     <main role="main" class="container">
 
-      <div class="starter-template">
-      <h1>Welcome</h1>
-      <h3>Something</h3>
-      <h3>Something</h3>
-      <h3>Something</h3>
-      </div>
+        <h2>Inventory Table</h2>
+  
+        <h4>Add New Inventory Item (make this a button)</h4>
+        <a href="/get-data"></a>
+
+        <table>
+          <tr>
+            <th>Item ID</th>
+            <th>Type</th>
+            <th>Description</th>
+            <th>Length</th>
+            <th>Width</th>
+            <th>Height</th>
+            <th>Scrap?</th>
+            <th>Last Update</th>
+            <th>Employee</th>
+          </tr>
+          <div>
+            {{# each items}}
```

Comment:
These lines are trash... I'm trying to implement a for each loop to populate the table from the database. It literally fills the table with "{{this.id}}" as a string...